fiware/src/Listener.hpp

```diff
@@ -18,6 +18,7 @@
 #ifndef SOSS__FIWARE__INTERNAL__LISTENER_HPP
 #define SOSS__FIWARE__INTERNAL__LISTENER_HPP
 
+#define ASIO_STANDALONE
```

Comment:
Maybe is more save adding this as a compiler definition (`target_compile_definitions` in `cmake`). If some user includes `asio.hpp` in other files, could not know the `ASIO_STANDALONE` requirement.